pages/common/xml-pyx.md

```diff
@@ -0,0 +1,16 @@
+# xml-pyx
+
+> Convert an XML document to PYX (ESIS - ISO 8879) format.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Convert an XML document to PYX format:
+
+`xml pyx {{path/to/input.xml|URI}} >{{path/to/output.pyx}}`
+
+- Convert an XML document from standard input to PYX format:
+
+`cat {{path/to/input.xml}} | xml pyx >{{path/to/output.pyx}}`
+
+- Display help for `pyx` subcommand:
```

Comment:
Perhaps this will read better?
```suggestion
- Display help for the `pyx` subcommand:
```

Reply:
There exists disagreement between reviewers regarding the need for "the", "an" and "a" relative to the desire for brevity. This could be resolved in the style guide.

Changed all places.

Reply:
xml.md:- Display help for a subcommand:
xml-canonic.md:- Display help for the `canonic` subcommand:
xml-depyx.md:- Display help for the `depyx` subcommand:
xml-edit.md:- Display help for the `edit` subcommand:
xml-elements.md:- Display help for the `elements` subcommand:
xml-escape.md:- Display help for the `escape` subcommand:
xml-format.md:- Display help for the `format` subcommand:
xml-list.md:- Display help for the `list` subcommand:
xml-pyx.md:- Display help for the `pyx` subcommand:
xml-select.md:- Display help for the `select` subcommand:
xml-transform.md:- Display help for the `transform` subcommand:
xml-unescape.md:- Display help for the `unescape` subcommand:
xml-validate.md:- Display help for the `validate` subcommand: